### ISSUE TITLE
Duping and deleting items in 2x2 grid

### ISSUE THREAD

**Issue by davqvist** [orbwoi/UniversalRemote#17]
MC 1.12.2
Forge 2512
UniversalRemote 0.9.6

So I updated my WIP pack from 1.12.1 to 1.12.2 and updated a whole bunch of mods but suddenly noticed something weird going on when crafting. Looked like a desync, but it was actually duping and later deleting items. I could narrow it down to only happen in the 2x2 inventory crafting grid. The hard part was to figure out what suddenly caused this. 3h later I noticed that updating Universal Remote from 0.9.4 to 0.9.6 caused that. So as it is hard to describe what actually goes on, here is a gif: 

https://cdn.discordapp.com/attachments/362326885784354818/370816576322273280/ashbug.gif

Basically no crafting recipe works in 2x2 and as soon as put stuff in it, you dupe items but after moving them around a bit they eventually get deleted. Not sure I can provide a better explanation.

**Comment by orbwoi** [comment]
That's.. very odd. Probably an artifact of the new Player hook system that was designed to help the mod not break across version, I suppose. I will look into this.

**Comment by orbwoi** [comment]
Fixed in 0.9.7
